kalendar-foundation/src/commonMain/kotlin/com/himanshoe/kalendar/foundation/component/KalendarDay.kt

```diff
@@ -58,7 +58,7 @@ fun KalendarDay(
     selectedDate: LocalDate = date,
     events: KalendarEvents = KalendarEvents(),
     dayKonfig: KalendarDayKonfig = KalendarDayKonfig.default(),
-    onDayClick: (LocalDate, List<KalendarEvent>) -> Unit = { _, _ -> },
+    onDayClick: (LocalDate, List<KalenderEvent>) -> Unit = { _, _ -> },
```

Comment:
I think you should extend compose compiler parameters and declare `LocalDate` as a stable type. This can help in perf

Reply:
Yea, can be done. Later in plan